core/error_macros.h

```diff
@@ -173,6 +196,18 @@ extern bool _err_error_exists;
 			_err_error_exists = false;                                                                     \
 	}
 
+/** Use this one if there is no sensible fallback, that is, the error is unrecoverable. 
+ */
+
+#define CRASH_COND(m_cond)                                                                                        \
+	{                                                                                                             \
+		if (m_cond) {                                                                                             \
+			_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "FATAL: Condition ' " _STR(m_cond) " ' is true."); \
+			GENERATE_TRAP                                                                                         \
+		} else                                                                                                    \
+			_err_error_exists = false;                                                                            \
```

Comment:
Since I wasn't getting how the flag is being used exactly, I just mimicked what ERR_COND does. But if you can confirm it's pointless, I will happily remove it. :)